lib/components/artist/articles/index.js

```diff
@@ -0,0 +1,48 @@
+/* @flow */
+'use strict';
+
+import Relay from 'react-relay';
+import React from 'react-native';
+const { ScrollView, StyleSheet, View } = React;
+
+import colors from '../../../../data/colors';
+import SerifText from '../../text/serif';
+import ImageView from '../../opaque_image_view';
+import Article from './article';
+
+class Articles extends React.Component {
+  render() {
+    const articles = this.props.articles
+    return (
+      <View style={styles.container}>
+        <SerifText style={styles.heading}>Featured Articles</SerifText>
+        <ScrollView horizontal={true} style={{ overflow: 'visible' }}>
+          { articles.map(article => <Article key={article.slug} article={article} />) }
```

Comment:
I think there’s a `key` prop provided by Relay or something http://facebook.github.io/react/docs/create-fragment.html

I also remember some React specific `map` helper that would generate keys, but not sure where/how.

I’m also thinking that maybe it just needs to be e.g. an index counter? Instead of fetching extra data.